jetty-server/src/test/java/org/eclipse/jetty/server/AsyncRequestReadTest.java

```diff
@@ -328,6 +328,7 @@ public void testPartialReadThenClose() throws Exception
 
             BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()));
             assertThat(in.readLine(), containsString("HTTP/1.1 200 OK"));
+            assertThat(in.readLine(), containsString("Connection: close"));
```

Comment:
Gah! This test will break as soon as we change the order of the headers, or if a JDK decides to change how things are stored in a `Map`. Not a biggie for this PR, but...

Reply:
Use our HttpTester here.

Reply:
yeah well it broke as soon as I changed our strategy for closing, so yes fragile.... but at least it made me look.

Reply:
Not keen on major rewrite of this test - there are lot of such tests.   We can fix them in another PR.